packet-ethereum-disc.c

```diff
@@ -336,8 +363,9 @@ static int process_pong_msg(tvbuff_t *packet_tvb,
 
   // Expiration.
   rlp_next(packet_tvb, rlp->next_offset, rlp);
-  proto_tree_add_item(packet_tree, hf_ethereum_disc_pong_expiration, packet_tvb,
-                      rlp->data_offset, rlp->byte_length, ENC_TIME_SECS | ENC_BIG_ENDIAN);
+  // FIXME: Disabled for now as Expiration on v5 pong is broken: https://github.com/ethereum/go-ethereum/issues/17468
+  // proto_tree_add_item(packet_tree, hf_ethereum_disc_pong_expiration, packet_tvb,
```

Comment:
It causes a `Malformed packet` error as it will attempt to read an absolute time from a too-short byte_length. I think the best option would be for us to try and catch that error here and set no expiration in those cases, but I'm not sure how to do that